innovation.game.php

```diff
@@ -9844,6 +9843,7 @@ function stPlayerInvolvedTurn() {
                     throw new EndOfGame();
                 }
                 foreach (self::getCardsInLocation($player_id, 'revealed') as $card) {
+                    $card = self::getCardInfo($card['id']);
```

Comment:
This bug doesn't actually happen when the card is in a publicly visible location (e.g. revealed or board). So let's remove this line so that we aren't making an unnecessary SQL query.